scenarios/Makefile

```diff
@@ -77,10 +94,37 @@ stc-dev-build:
 	cd /tmp/onos-stc && mvn -q -B -Dmaven.test.skip=true install
 
 pull: ./tmp/.env.docker
+ifeq ($(BUILD_SDFABRIC),true)
+	@#avoid pulling the ONOS image if we build it locally
+	${DOCKER_COMPOSE_CMD} pull pfcp-agent mock-smf p4rt mininet atomix1 atomix2 atomix3
+else
 	${DOCKER_COMPOSE_CMD} pull
+endif
 
 build: ./tmp/.env.docker
+ifeq ($(BUILD_SDFABRIC),true)
+	@#avoid pulling the ONOS image if we build it locally
+	${DOCKER_COMPOSE_CMD} build --pull pfcp-agent mock-smf p4rt mininet atomix1 atomix2 atomix3
+else
 	${DOCKER_COMPOSE_CMD} build --pull
+endif
+
+build-sdfabric: ./tmp/.env.docker sdfabric-onos
+ifeq ($(BUILD_SDFABRIC),true)
+	@export $(shell cat "${CURRENT_DIR}"/tmp/.env.docker | grep -v '#' | xargs) && \
+	cd sdfabric-onos && git pull origin master && \
+	USE_LOCAL_SNAPSHOT_ARTIFACTS=true DOCKER_TAG=${SDFABRIC_DOCKER_TAG} DOCKER_REPOSITORY=sdfabric-onos make docker-build JOBS=${NPROCS}
+	#USE_LOCAL_SNAPSHOT_ARTIFACTS=true DOCKER_TAG=${SDFABRIC_DOCKER_TAG} DOCKER_REPOSITORY=sdfabric-onos make package JOBS=${NPROCS}
+	@sed -i '' -e 's/ONOS_IMAGE=.*/ONOS_IMAGE=sdfabric-onos:master/' "${CURRENT_DIR}"/tmp/.env.docker
+	@sed -i '' -e 's/ONOS_IMAGE=.*/ONOS_IMAGE=sdfabric-onos:master/' "${CURRENT_DIR}"/../.env.${UP4_ENV}
```

Comment:
I don't like to change the actual env file, however I don't think there is another way